test/e2e/support/setup-test-framework.js

```diff
@@ -55,8 +52,11 @@ afterAll( async () => {
 		await page.select( '#bulk-action-selector-top', 'trash' );
 		// Submit the form to send all draft/scheduled/published posts to the trash.
 		await page.click( '#doaction' );
-		await page.waitForNavigation();
+		await page.waitForXPath(
+			'//*[contains(@class, "updated notice")]/p[contains(text(), "moved to the Trash.")]'
+		);
 	}
 
-	await browser.close();
+	disablePageDialogAccept();
+	await clearLocalStorage();
```

Comment:
This seems like something we ought to do before tests start, since `afterAll` might not be run if the test process is terminated abruptly, and would thus leave the browser in a tampered state for the next run?

Reply:
Let's try to move it to `beforeEach` to ensure all tests start with the clean state. We can move to `beforeAll` if that turns out to be too aggressive.